### PR TITLE
(DOCSP-19704) Install

### PR DESCRIPTION
**Pull request by biniona-mongodb**
## Pull Request Info

Add install page

### Issue JIRA link:

https://jira.mongodb.org/browse/DOCSP-19704

### Snooty build log:

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=61c0cf25c52f2a460f61ae8c

### Docs staging link (requires sign-in on MongoDB Corp SSO):

https://docs-mongodbcom-staging.corp.mongodb.com/visual-studio-extension/docsworker-xlarge/DOCSP-19704-Install/install/

### Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Does it render on staging correctly?
- [ ] Are all the links working? **NuGet package link is broken as the package has not been added yet**
- [x] Are the staging and workerpool job links in the PR description updated?

### If your page documents a concept, does it meet the following criteria?

- [ ] Target the [Jasmin persona](https://drive.google.com/file/d/14FbBOLCVxwSP6M9BK4Nz1Ir9tzxT8_02/view)
- [ ] Target the [Lucas persona](https://drive.google.com/file/d/1J2vqJxo7ldv7OP_obA9Q-avf0o_ju4Lk/view)

### CODE REVIEW

File: source/includes/nuget-versioned-install/1.0.0.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.. code-block:: shell
+
+   Install-Package MongoDB.Analyzer -Version 1.0.0
```
Comment (terakilobyte) [inline source/includes/nuget-versioned-install/1.0.0.rst:3]: question: Why define constants if you aren't using them here? It looks like you may have meant to?

**Review by terakilobyte — APPROVED** [review]
LGTM with one question to answer please.

File: source/includes/nuget-versioned-install/1.0.0.rst (hunk 1 above)
Comment (biniona-mongodb) [reply source/includes/nuget-versioned-install/1.0.0.rst]: I was under the impression that constants were not replaced in code blocks but this seems to not be the case. Good catch!

Removed includes and replaced with code snippet that uses constants.

**Review by BorisDog — APPROVED** [review]
LGTM

**Review by mmarcon — APPROVED** [review]
LGTM. Is the version number required or if you don't specify it the most recent version will be installed?

**Comment by biniona-mongodb** [comment]
@mmarcon, based [off of this documentation](https://docs.microsoft.com/en-us/nuget/reference/ps-reference/ps-ref-install-package#parameters) it seems like it defaults to the latest version if you do not specify the version.

It seems like the [NuGet gallery generally specifies the version](https://www.nuget.org/packages/MongoDB.Driver/) when providing an install-package command, so it seemed appropriate to do it here as well. I agree though this could raise maintenance concerns if we accidentally recommend installing a dated version.

I think it makes sense to keep it as is for now (as this seems to closely resemble installation on NuGet while not making readers have to engage with the NuGet site), and if we run into issues in the future we do one of the following:

- Remove the version
- Not provide any installation code snipped and only link to NuGet.

@BorisDog and @mmarcon feel free to let me know if you believe my thinking is off on how best to handle the Connector version in this code snippet.

**Pull request merged**